styles/layout.css

```diff
@@ -758,20 +758,21 @@ table.dirlist caption {
 #toolbox .char_box {
   padding-top: 1px;
 }
+#toolbox .big_text,
+#toolbox .picker {
+  font-family: DejaVu Sans Mono;
+  vertical-align: middle;
+  text-align: center;
+  width: 1.1em;
+}
 #toolbox .big_text {
   font-size: 300%;
-  width: 1.3em;
   margin: 2px;
-  text-align: center;
 }
 #toolbox .picker {
-  margin-top: 1px;
   font-size: 1em;
+  margin-top: 1px;
   padding: 0 1px 0 0;
-  vertical-align: middle;
-  width: 100%;
-  min-width: 1em;
-  max-width: 1.1em;
```

Comment:
This width stuff was put in to try to make it work with some old browsers. My internet is so bad now I can't use browserstack to look at it again.

Reply:
That's an excellent question Ray, thanks for bring it up as I had completely forgotten about BrowserStack. I tested it against our list of [Supported Browsers](https://www.pgdp.net/wiki/DP_Official_Documentation:General/Supported_Browsers) and the picker itself actually fails at what we claim to support (oops).

I did do testing and we need to adjust our Support Browser set to be the following:
* Chrome 50
* Firefox 50
* Opera 40
* Safari 10

Reply:
And to be clear - this isn't because of this change, but because we were just wrong in what browsers at least the picker itself worked on.

Reply:
I will update our supported browsers page and have filed a bug to create a manual test plan and do some more thorough validation.